LayoutTests/platform/wpe/TestExpectations

```diff
@@ -538,6 +538,10 @@ webkit.org/b/244186 imported/w3c/web-platform-tests/event-timing/large-duration-
 webkit.org/b/244186 imported/w3c/web-platform-tests/event-timing/medium-duration-threshold.html [ Skip ]
 webkit.org/b/244186 imported/w3c/web-platform-tests/event-timing/retrievability.html [ Skip ]
 
+# Event timing: timing out for unclear reasons (scheduling?). Events handlers are called but event timing entries aren't produced
+webkit.org/b/214677 imported/w3c/web-platform-tests/event-timing/gap-keydown-keyup.html [ Skip ]
```

Comment:
Definitely worth a separate bugzilla for other ports to evaluate when and if they want to enable this feature.